.gitignore

```diff
@@ -33,3 +33,5 @@ imgui.ini
 # but can be excluded explicitly
 demos/*
 tmp/*
+
+examples/tutorials/demos
```

Comment:
I suggest you create a new .gitignore under `examples/tutorials` instead of modifying the .gitignore under the root directory.